src/Components/Layout.tsx

```diff
@@ -60,15 +81,14 @@ export function Layout() {
     }
   };
 
-  function loadDates(dates) {
-    return JSON.parse(JSON.stringify(dates, null, 2));
-  }
-
-  function initDates() {
-    if (start) {
-      setStart(false);
-      setSelected([dates[0]["day"] - 1, dates[1]["day"] - 1]);
-    }
+  function formatDateforDisplay(index) {

```

Comment:
Can this be pulled out into a utility function so it's not duplicated between this file and body.tsx?

Reply:
it didnt need to be in body.tsx, and it is used in layout, fixed.